### PR TITLE
Added example for layout block

### PR DESCRIPTION
**Pull request by chandru-199**
## Purpose of this pull request

This pull request (PR) - Added example code snippet for layout block.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/xml-instructions.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 1]
```diff
@@ -74,6 +74,14 @@ We recommend always adding a `name` to blocks. Otherwise, it is given a random n
 
 To pass parameters use the [`<argument></argument>`](#argument) instruction.
 
+Sample of usage in layout:
+
+```xml
+<block class="Magento\Catalog\Block\Product\ListProduct" name="category.products.list" as="product_list" template="Magento_Catalog::product/list.phtml"/>
+```
+
+This would used in product listing page layout.
```
Comment (jeff-matthews) [inline src/guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion

```

File: src/guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md [hunk 2]
```diff
@@ -74,6 +74,14 @@ We recommend always adding a `name` to blocks. Otherwise, it is given a random n
 
 To pass parameters use the [`<argument></argument>`](#argument) instruction.
 
+Sample of usage in layout:
```
Comment (jeff-matthews) [inline src/guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md]:
```suggestion
Sample of usage in the product listing page layout:
```

**Comment by chandru-199** [comment]
@jeff-matthews I think it has more clarity while reading. If you strongly recommends, I will make that changes also.

**Comment by jeff-matthews** [comment]
@chandru-199, yes, please make the change.

**Comment by chandru-199** [comment]
@jeff-matthews Changes made as per your advice.Thanks.

**Pull request merged**